readme.md

```diff
@@ -1,4 +1,11 @@
 # Sequelize ERD
+![](https://img.shields.io/github/languages/top/wwmoraes/sequelize-erd.svg)
+![](https://img.shields.io/github/repo-size/wwmoraes/sequelize-erd.svg)
+![](https://img.shields.io/github/issues/wwmoraes/sequelize-erd.svg)
+![](https://img.shields.io/david/wwmoraes/sequelize-erd.svg)
+![](https://img.shields.io/github/last-commit/wwmoraes/sequelize-erd.svg)
```

Comment:
I wish there was a better way to do these shields with forked projects that are going to become PRs...